Snakefile

```diff
@@ -416,6 +415,20 @@ if config["enable"].get("retrieve_cost_data", True):
         run:
             move(input[0], output[0])
 
+    rule retrieve_cost_data_flexible:
```

Comment:
I assume this rule was added since in PyPSA-Earth the year was retrieved from the config, whereas in PyPSA-Earth-Sec it was retrieved from the wildcard `planning_horizons`. We should align this in general, in my point of view the wildcard `planning_horizons` makes sense.

Reply:
yes, I aggree. the problem is that the `planning_horizon` wildcard is not esistent for the power-only system. therefore, for now we should keep the distinction to avoid breaking changes.